contracts/BinaryOptionMarket.sol

```diff
@@ -49,99 +42,77 @@ contract BinaryOptionMarket is Owned, MixinResolver, IBinaryOptionMarket {
     /* ========== STATE VARIABLES ========== */
 
     Options public options;
-    Prices public prices;
     Times public times;
     OracleDetails public oracleDetails;
     BinaryOptionMarketManager.Fees public fees;
-    BinaryOptionMarketManager.CreatorLimits public creatorLimits;
+    IAddressResolver public resolver;
 
-    // `deposited` tracks the sum of open bids on short and long, plus withheld refund fees.
+    // `deposited` tracks the sum of all deposits minus the withheld fees.
     // This must explicitly be kept, in case tokens are transferred to the contract directly.
     uint public deposited;
+    uint public initialMint;
```

Comment:
Creator will not pay the fee for initial mint, so we keep the variable for later.